src/util/mongo.controller.js

```diff
@@ -120,6 +120,15 @@ const getBulkExportStatus = async clientId => {
   return status;
 };
 
+/**
+ * Wrapper for the updateResource function that updates the bulk status
+ * @param {*} clientId The id signifying the bulk status request
+ * @param {*} newStatus
+ */
+const updateBulkExportStatus = async (clientId, newStatus) => {
+  updateResource(clientId, { status: newStatus }, 'bulkExportStatuses');
```

Comment:
```suggestion
  await updateResource(clientId, { status: newStatus }, 'bulkExportStatuses');
```

I think we need this since we do care about waiting until the DB operation finished before returning from here